stabilize.md

```diff
@@ -340,27 +340,127 @@ Vow Semantics
  // -----------------------------------------------------------------
     rule <k> Vow . _ => exception ... </k> [owise]
 
-    syntax VowAuthStep ::= "fess" Int
+    syntax VowAuthStep ::= "fess" Rad
  // ---------------------------------
+    rule <k> Vow . fess TAB => . ... </k>
+         <currentTime> NOW </currentTime>
+         <vow-sins>
+           ...
+           NOW |-> (SIN' => SIN' +Rat TAB)
+           ...
+         </vow-sins>
+         <vow-sin> SIN => SIN +Rat TAB </vow-sin>
 
     syntax VowStep ::= "flog" Int
  // -----------------------------
+    rule <k> Vow . flog ERA => . ... </k>
+         <currentTime> NOW </currentTime>
+         <vow-wait> WAIT </vow-wait>
+         <vow-sins>... ERA |-> (SIN' => 0) </vow-sins>
+         <vow-sin> SIN => SIN -Rat SIN' </vow-sin>
+      requires ERA +Int WAIT <=Int NOW
 
     syntax VowStep ::= "heal" Rad
  // -----------------------------
+    rule <k> Vow . heal AMOUNT
+          => call Vat . heal AMOUNT ... </k>
+         <this> THIS </this>
+         <vat-dai>
+           ...
+           THIS |-> DAI
+           ...
+         </vat-dai>
+         <vat-sin>
+           ...
+           THIS |-> VATSIN
+           ...
+         </vat-sin>
+         <vow-sin> SIN </vow-sin>
+         <vow-ash> ASH </vow-ash>
+      requires AMOUNT <=Rat DAI
+       andBool AMOUNT <=Rat VATSIN -Rat SIN -Rat ASH
+       andBool VATSIN >=Rat SIN +Rat ASH
```

Comment:
Where is this last condition coming from? I don't see it here: https://github.com/makerdao/dss/blob/407afdcb034db841b25823a4aa312c005b3edf4d/src/vow.sol#L110

Reply:
This is implied by using safemath `sub`